integration_tests/tests/consistency/consistency_project_enhanced.sql

```diff
@@ -0,0 +1,48 @@
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+{% set exclude_columns = ['avg_age_currently_open_seconds', 'avg_age_currently_open_assigned_seconds', 'median_age_currently_open_seconds', 'median_age_currently_open_assigned_seconds', 'epics', 'components'] %}
```

Comment:
exclude cols that operate in current seconds to current time since it will be different between runs and cols that agg strings in a random order.

Reply:
@fivetran-catfritz could you add this explanation as a comment, so anyone running the test in the future knows to keep these excluded?

Reply:
does this apply to epics and components though?

Reply:
Yes, it would still apply since the string_agg assembles the strings in a random order each time.